locales/config.yaml

```diff
@@ -0,0 +1,26 @@
+---
+# This is the project-specific configuration file for setting up
+# fast_gettext for your project.
+gettext:
+  # This is used for the name of the .pot and .po files; they will be
+  # called <project_name>.pot?
+  project_name: puppetlabs-registry
+  # This is used in comments in the .pot and .po files to indicate what
+  # project the files belong to and should bea little more desctiptive than
+  # <project_name>
+  package_name: puppetlabs-registry
+  # The locale that the default messages in the .pot file are in
+  default_locale: en
+  # The email used for sending bug reports.
+  bugs_address: docs@puppet.com
+  # The holder of the copyright.
+  copyright_holder: Puppet, Inc.
+  # This determines which comments in code should be eligible for translation.
+  # Any comments that start with this string will be externalized. (Leave
+  # empty to include all.)
+  comments_tag: TRANSLATOR
+  # Patterns for +Dir.glob+ used to find all files that might contain
+  # translatable content, relative to the project root directory
+  source_files:
+      - 'metadata.json'
+  
```

Comment:
@glennsarti this was added in [this commit](https://github.com/puppetlabs/modulesync_configs/commit/e25ca94cb29cf567861000f93e8054f1508a80de) to modulesync_configs by @eputnam and as such if I edit this PR to add a newline it will be overwritten in the next msync run. Do you want this amended in the modulesync configs repo and rolled out in the next msync run with the newline?